en_US/dashboard/rules.md

```diff
@@ -53,43 +57,32 @@ For other events, we can select different client and session events to simulate
 - Subscribed ($events/session_subscribed)
 - Unsubscribed ($events/session_unsubscribed)
 
-There is also a way to use data bridges as data sources, using the `$bridges/*` topic to represent the message or event of the data bridge:
-
-- Data Bridge ($bridge/\<type>:\<name>)
-
-The corresponding data source needs to be consistent with the SQL statement in the editor. When you need to use some message events or data bridges to get data, you need to fill in the corresponding event topic (the content in the brackets) into the SQL statement after the `FROM` keyword, and rules support the use of multiple events. For more details about data source and event explanations, please refer to [SQL Data Source and Fields](../data-integration/rule-sql-events-and-fields.md).
-
-![image](./assets/rule-test.png)
-
-### Actions
-
-After completing the editing of the SQL statement and the rule debugging, we can get output data that meets our requirements. At this time, we can select the action that needs to be executed after the rule is triggered in the `Actions` tab on the right side of the page. Click the `Add Action` button, and we can choose two built-in actions or use data bridging to continue to process the output data of the rule. The two built-in actions include:
+The corresponding data source needs to be consistent with the SQL statement in the editor. When you need to use some message events to get data, you need to fill in the corresponding event topic (the content in the brackets) into the SQL statement after the `FROM` keyword, and rules support the use of multiple events. For more details about data source and event explanations, see [SQL Data Source and Fields](../data-integration/rule-sql-events-and-fields.md).
 
-1. Console Output, which can print the resulting output to the console, can be used for debugging in the real scene;
-2. Republish, which can republish the output data of the rule processing to the specified topic again.
+You can also test the rule in the testing area. For detailed testing steps, you can see [Test Rule](../data-integration/rule-get-started.md#test-rule) for details.
```

Comment:
Is this duplicated?  The current section seems to be about rule testing already.

Reply:
There are two functions with two separate tabs under the "Try It Out" in Dashboard now: SQL testing and Rule testing.